databook/glossary.md

```diff
@@ -20,7 +20,8 @@ Cre line
   experiments, Cre is used with loxP {term}`Reporter line` in order to drive
   recombinase of the loxP sites and drive the expression of the reporter. As Cre
   is often expressed within a specific gene, this allows the reporter expression
-  to be restricted to particular subset of cells.
+  to be restricted to particular subset of cells. For specific lines used, see
+  the section on [transgenic tools](transgenic-tools).
 
 CSV
```

Comment:
can we remove CSV? I don't think this belongs in here.

Reply:
CSV removed from glossary (also added action potential, which was not there previously)